src/app/components/ConsentBanner/index.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string } from 'prop-types';
+import { GEL_GREAT_PRIMER, GEL_PICA } from '@bbc/gel-foundations/typography';
+import { C_WHITE } from '@bbc/psammead-styles/colours';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { FF_NEWS_SANS_REG } from '@bbc/psammead-styles/fonts';
+
+const mainBackgroundColor = '#323232';
+const buttonColor = '#f6a21d';
+const paragraphColor = '#bebebe';
+
+const Prompt = styled.div`
+  background-color: ${mainBackgroundColor};
+  padding: ${GEL_SPACING_DBL};
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    &:after {
+      content: \\0020;
```

Comment:
I appreciate this is copied from Alephamp, but might be good to document why this exists